src/Depends.Core/DependencyAnalyzer.cs

```diff
@@ -297,6 +297,14 @@ private DependencyGraph.Builder CreateBuilder(IProjectAnalyzer  projectAnalyzer,
                     //}
                 }
 
+                // Not all dependencies are necessarily included in the list of libraries
+                // for the current target framework and runtime identifier. Add these to libraryNodes
+                // so we can still record these dependencies.
+                foreach (var dependency in libraries.SelectMany(library => library.Dependencies))
+                {
+                    libraryNodes.TryAdd(dependency.Id, new PackageReferenceNode(dependency.Id, ""));
```

Comment:
Can we get the version of the dependency? Add empty string as version is not optimal

Reply:
So, that one's a bit tricky. All we have here is a PackageDependency object. This does not have a single version, but a version range (i.e. "at least 2.2.0").

That's not a problem by itself, but it does raise the question of how to handle cases where A and B both depend on C, but they have different version ranges.

You'd have to go through all of the libraries that appear only as dependencies, combine their version ranges, and only then add them to libraryNodes. It's an extra step and it would mean not being able to do everything in one iteration as you requested in the other comment. It's a bit more complicated and it doesn't really add anything useful for the user. "Which libraries depend on this node and with what version ranges" can already be answered by looking at the "Reverse Depends" window, after all.